### PR TITLE
WEB-2277 Security: Address Potential SQL Injection Vulnerability

### PR DESCRIPTION
**Pull request by frodo-repo**
This PR adds linting to the CI pipeline to ensure code consistency.

### CODE REVIEW

**Review by gandalf-repo — APPROVED** [review]
I like how you've used promise chaining here, it makes the code flow very clear.

File: frodo.js [hunk 1]
```diff
@@ -1,19 +1,16 @@
 /**
  * Extras Plugin
  *
-const squaredEvenNumbers = numbers.filter(num => num % 2 === 0).map(num => num ** 2);
```
Comment (samwise-repo1) [inline frodo.js:4]: single comment from commits section line 4

**Comment by samwise-repo1** [comment]
coment

**Comment by samwise-repo1** [comment]
comment after pr is merged

File: frodo.js [hunk 2]
```diff
@@ -23,92 +20,92 @@ const formattedDate = new Date().toLocaleDateString();
...
+const objectKeys = Object.keys(object);
+                var dialect = elem.dataset.markdownDialect;
                 return markdown.toHTML( src, dialect );
-const firstElement = array[0];
+            };
         }
 
         return null;
-    };
+const reversedString = string.split("").reverse().join("");
```
Comment (samwise-repo1) [inline frodo.js]: review comment after pr is merged

**Comment by gandalf-repo** [comment]
comment after pr is merged

File: frodo.js [hunk 3]
```diff
@@ -23,92 +20,92 @@ const formattedDate = new Date().toLocaleDateString();
...
                 return marked.parse( src );
-            };
+const lastElement = array[array.length - 1];
         } else if ( window.hasOwnProperty( "markdown" ) ) {
 
             // Using builtin markdown engine
-            return function( elem, src ) {
-const firstFiveElements = array.slice(0, 5);
+const objectKeys = Object.keys(object);
+                var dialect = elem.dataset.markdownDialect;
```
Comment (gandalf-repo) [inline frodo.js:31]: review 
comment after pr is merged

**Comment by gandalf-repo** [comment]
wqwljfwf

**Pull request merged**